db/featured.ts

```diff
@@ -17,7 +17,7 @@ export const FEATURED_PROJECTS = [
   {
     color: '#0280BE',
     logo: 'rancher-desktop.svg',
-    tag: 'storage',
+    tag: 'Development',
```

Comment:
might need to be lowercased according to https://github.com/SUSE/project-portal/blob/1e5f82439bfdd61d14f268c5a4c90a9c683f7dd8/db/projects.ts#L54

Reply:
You are right, I grabbed from the website, and I didn't realized it was lowercase